### PR TITLE
Removes eClock from C++ examples [6316]

### PR DESCRIPTION
**Pull request by raquelalvarezbanos**
This PR removes usage of eClock from our C++ examples. To test this PR:

* Do a grep and make sure that eClock is not included/used in any example
* Check a few examples and make sure they still work

### CODE REVIEW

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/381/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/347/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/379/)

File: examples/C++/Benchmark/BenchmarkSubscriber.cpp [hunk 1]
```diff
@@ -116,43 +111,6 @@ bool BenchMarkSubscriber::init(int transport, ReliabilityQosPolicyKind reliabili
...
-            Rparam.topic.topicDataType = "Dyn_BenchMarkBig";
-            Wparam.topic.topicDataType = "Dyn_BenchMarkBig";
-            m_dynType.SetDynamicType(m_DynamicData);
-            Domain::registerType(mp_participant, &m_dynType);
-            break;
-        }
-        m_DynamicData->set_uint32_value(0, 0);
-    }
-    else*/
     {
```
[the remark this answers is not in the document]
Comment (raquelalvarezbanos) [reply examples/C++/Benchmark/BenchmarkSubscriber.cpp]: Changed

File: examples/C++/ClientServerTest/EprosimaServer.cpp [hunk 2]
```diff
@@ -63,7 +63,9 @@ void EprosimaServer::serve()
 void EprosimaServer::serve(uint32_t samples)
 {
 	while(m_n_served < samples)
```
[the remark this answers is not in the document]
Comment (raquelalvarezbanos) [reply examples/C++/ClientServerTest/EprosimaServer.cpp]: Indented whole file

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/386/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/352/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/384/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/389/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/355/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/387/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/394/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/360/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/392/)

**Comment by raquelalvarezbanos** [comment]
Failing test is SimpleComunicationSecureBestEffort.

**Pull request merged**